clevertap-core/src/main/java/com/clevertap/android/sdk/inapp/InAppController.java

```diff
@@ -74,7 +76,7 @@ private final class NotificationPrepareRunnable implements Runnable {
 
         private JSONObject jsonObject;
 
-        private final boolean videoSupport = Utils.haveVideoPlayerSupport;
+        private final boolean videoSupport = VideoLibChecker.INSTANCE.haveVideoPlayerSupport;
```

Comment:
Why not VideoLibChecker.haveVideoPlayerSupport instead of VideoLibChecker.INSTANCE.haveVideoPlayerSupport